docker/php.ini

```diff
@@ -1 +1,2 @@
 extension=mongodb.so
+memory_limit=2G
```

Comment:
Is this because of Composer?

Reply:
It's needed for mongo

Reply:
I can't justify this line.  It came from another project and was in a different .ini.  I consolidated it into php.ini.  
Local tests pass; I'll remove it and run travis

Reply:
memory_limit has been removed